.drone.yml

```diff
@@ -0,0 +1,54 @@
+kind: pipeline
+name: Build prospero
+type: kubernetes
+
+
+steps:
+- name: build 
+  image: plugins/docker
+  settings:
+    username: 
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+    repo: maxsurm/prospero
```

Comment:
From fork? :)

Reply:
insert your credentials to Drone secrets. Or we can try build and publish docker image in Github Action, and deploy via Drone CI

Reply:
I've set up GH Actions to build docker image and publish  it to the [Docker Hub](https://hub.docker.com/repository/docker/kinkard/prospero)  - https://github.com/kinkard/prospero/blob/main/.github/workflows/push_docker_image.yml

Reply:
Okay, maybe you're need a help with autodeploy?
And I think this PR not relevant

Reply:
Decently yes